.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: build
+
+on:
```

Comment:
Not sure what's intended here. This will currently run on all pull requests and also all branches that don't contain a `/`. (Since `*` [matches](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) all characters except `/`). Also I don't know what `{}` does here. I think what we want is to run on all pull requests and every commit to master. That would look like:
```
  pull_request:
  push:
    branches:
      - master
```

Reply:
definitely. Thanks!